integration/file.test.js

```diff
@@ -0,0 +1,36 @@
+const { SkynetClient, defaultPortalUrl, uriSkynetPrefix } = require("../index");
+const portalUrl = defaultPortalUrl();
+const client = new SkynetClient(portalUrl);
+
+const newTimeout = 60000;
+jest.setTimeout(newTimeout);
+jest.useRealTimers();
```

Comment:
What is this for? 🤔

Reply:
hello m-cat,

The lines are from before RP on the master updating.
I can take it out and see if it works.